docs/README.md

```diff
@@ -865,7 +865,87 @@ _Reference_
 - https://radiant-brushlands-42789.herokuapp.com/medium.com/better-programming/javascript-es2020-features-with-simple-examples-d301dbef2c37
 - https://www.freecodecamp.org/news/javascript-new-features-es2020/
 
-<br/> <br/> <br/>
+<br/>
+
+## 17. DOM Manipulation
+
+Document Object Model, or DOM for short, represents all page content as objects that can be modified. And Javascript allow us to modify DOM.
+
+With JavaScript you can interact with the DOM to:
```

Comment:
I think this line 874-881 is not required
Let's keep the file as simple as possible

Reply:
Done.